src/components/Trade/Components/TradeQuotes/TradeQuote.tsx

```diff
@@ -91,30 +138,24 @@ export const TradeQuoteLoaded: React.FC<TradeQuoteLoadedProps> = ({
       px={4}
       py={2}
       fontSize='sm'
-      onClick={() => onClick(protocol)}
+      onClick={() => handleSwapperSelection(swapperWithMetadata)}
```

Comment:
Non-blocking but if we want to avoid a stack trace, `handleSwapperSelection` might benefit from not being pure, since `swapperWithMetadata` is reactive. `handleSwapperSelection` would have to be callback-memoized however.

Unless it being pure and not reactive is actually the intent, in which case we should probably still `useCallback()` with no deps, it and silence the linter with a comment explaining why.

Reply:
Resolved in https://github.com/shapeshift/web/pull/4079/commits/0543329e80fb6c0944b4790d45acc67db58a2064

Non-reactive was not the intent! Good catch.